EpicLoot/Adventure/BountyTarget.cs

```diff
@@ -51,17 +51,25 @@ private bool HasBeenSetup()
         private void OnDeath()
         {
             EpicLoot.LogWarning("BountyTarget.OnDeath");
-            var pkg = new ZPackage();
-            _bountyInfo.ToPackage(pkg);
+            if (ZNet.instance.IsServer() || !ZNet.instance.IsServer() && !ZNet.instance.IsDedicated())
+            {
+                var pkg = new ZPackage();
+                _bountyInfo.ToPackage(pkg);
 
-            EpicLoot.LogWarning($"SENDING -> RPC_SlayBountyTarget: {_monsterID} ({(_isAdd ? "minion" : "target")})");
-            ZRoutedRpc.instance.InvokeRoutedRPC("SlayBountyTarget", pkg, _monsterID, _isAdd);
+                EpicLoot.LogWarning($"SENDING -> RPC_SlayBountyTarget: {_monsterID} ({(_isAdd ? "minion" : "target")})");
+                ZRoutedRpc.instance.InvokeRoutedRPC("SlayBountyTarget", _monsterID, _isAdd, pkg);
```

Comment:
Bug I encountered in testing here, you switched the order of the parameters for this. Was there a reason why?

Reply:
Sorry, was away for a bit.  I had originally tried a default null parameter for pkg so I put it last, I must have forgotten to put it back when I did my code clean-up.